apps/demos/Demos/HtmlEditor/MarkdownSupport/Angular/app/app.component.css

```diff
@@ -3,12 +3,19 @@
   padding-right: 10px;
 }
 
+::ng-deep .value-title {
+  font-size: 18px;
```

Comment:
Can we use any from predefined css variables for font for this purpose? e.g. `--dx-font-size-sm`

Reply:
[fb5ade491777215a063dad2c9b8f6de94f502853]